content/3_Silver/Binary_Search_Ans.mdx

```diff
@@ -649,7 +664,7 @@ def search():
 		a //= 2
     return int(pos)
 
-v=[]
+v=['Binary Search']
```

Comment:
here

Reply:
just commenting on this one time is enough :)